SetupGuide_Plugin_Admin.php

```diff
@@ -94,8 +94,8 @@ public function skip() {
 	 * @uses $_POST['choice'] TOS choice: accept/decline.
 	 */
 	public function set_tos_choice() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
-			$choice          = empty( $_POST['choice'] ) ? null : sanitize_key( $_POST['choice'] );
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the phpcs comment to have it validate the change.

Reply:
Removed PHPCS disable comment.